liquidctl-completion.sh

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env bash
```

Comment:
Let's move this to `extra/linux`.

Reply:
This wouldn't be specific to linux tho, and would be installed as part of the dist package

Reply:
Regarding distribution, everything in `extra` is part of the sdist.

But you're right, this is not specific to Linux (although also not super general, as many users are not running bash, even on *nix platforms).  So how about `extra/bash/liquidctl-completion.sh`?

_(We'll presumably have other completion scripts in the future, and this one is specific to bash.)_

Reply:
how about `extra/completions/liquidctl-completion.sh`? since there likely would be more just bash extras

Reply:
I think `bash` has to be somewhere in the path so it wont be confusing when another completion script is added.

But at this point it doesn't matter _that_ much, it's just something to remember to check/fix before 1.5.0.

Reply:
Naming things is hard, but also important.  But this pattern (shell completions) is very common, how are other projects handling the multiple shells?  We should the same, just (somewhere) in `extra/`.

Reply:
> This pattern (shell completions) is very common, how are other projects handling the multiple shells

git: 
`contrib/completion/git-completion.bash`

Most of the completions that I have found have been located in the one `bash-completions` project and not in the individual project that they belong to. (tbh I also didnt look all that hard for individual projects with them)